lib/monotonic_tick_count.rb

```diff
@@ -53,5 +53,11 @@ class << self
     def now
       new(tick_count_f: Process.clock_gettime(Process::CLOCK_MONOTONIC))
     end
+
+    def timer
+      start = MonotonicTickCount.now
+      result = yield
+      [result, MonotonicTickCount.now - start]
```

Comment:
I wonder if this interface will be convenient? A variation would be to not return the result at all. I think that's what `Benchmark.measure` does.

Reply:
I was inspired to add it from `Invoca::Metrics::Client#time` which returns a pair. The use cases in ringswitch where I plan to use it expect to use the result. For example:
https://github.com/Invoca/web/blob/183214c84d9bb0a75eb7d9c7c3f81da684f6b16e/lib/ringswitch/filter_bundled_destination_ids_publishing.rb#L52

I imagined, in the case that the result was not needed, it can be ignored with `_, duration = MonotonicTickCount.timer { ... }`. I'm open to returning only the result, but I'd prefer to return both. Let me know if the comment I've added is sufficient.